cartographer_grpc/framework/server_test.cc

```diff
@@ -51,6 +53,29 @@ class GetSumHandler
   int sum_ = 0;
 };
 
+class GetRunningSumHandler
+    : public RpcHandler<Stream<proto::GetSumRequest>, Stream<proto::GetSumResponse>> {
+ public:
+  void OnRequest(const proto::GetSumRequest& request) override {
+    sum_ += request.input();
+
+    auto response = cartographer::common::make_unique<proto::GetSumResponse>();
```

Comment:
// Respond twice to demonstrate bidirectional streaming.

Reply:
Added.